versions-ecosystem.cfg

```diff
@@ -0,0 +1,22 @@
+# These are extra version pins for packages that are not part of core or tests.
+# But they are often used modules and we want to have canonical versions of them.
+# Note: this is ony lightly tested, and we cannot give guarantees that these versions work without error.
+# Read the documentation of the individual packages, or ask the maintainers.
+[versions]
+# collective.z3cform.datagridfield = 2.0.2
+# collective.z3cform.datetimewidget = 1.2.9
```

Comment:
you kept these pins commented as they are not needed, but might be worth keeping them around as we might want to uncomment them eventually? 🤔

Reply:
I kept them because it would be good to have version pins for these, as they were in the 5.2 pins as well. But there are no released versions of these two packages yet that work on current Plone 6.
Some work is done here:

- https://github.com/collective/collective.z3cform.datetimewidget/pull/12
- https://github.com/collective/collective.z3cform.datagridfield/pull/120